interactive/commands.py

```diff
@@ -0,0 +1,65 @@
+from jobserver.authorization import InteractiveReporter
+from jobserver.github import _get_github_api
+from jobserver.models import OrgMembership, ProjectMembership, Repo, User
+
+
+def create_repo(*, name, get_github_api=_get_github_api):
+    """
+    Create an interactive GitHub repo
+
+    The repo might already exist so we try to get it first, and then we ensure
+    it has the correct topics set.
+    """
+    api = get_github_api()
+
+    repo = api.get_repo("opensafely", name)
+
+    if repo is None:
+        repo = api.create_repo("opensafely", name)
+
+    api.set_repo_topics("opensafely", name, ["interactive"])
+
```

Comment:
I've created an [Interactive](https://github.com/orgs/opensafely/teams/interactive) team as a way to group interactive repos with `write` permission?. Do you think we could add this newly created repo to that team?

I think topics are informative and useful for searching. But adding them to a team means we can control privileges better.

e.g. we could give the interactive bot account write access to just the repos on this team.

Reply:
Absolutely lets add it to the team 👍